integration/sinatra1_app/app.rb

```diff
@@ -71,11 +70,21 @@ def trigger_breakpoint
   "breakpoint triggered"
 end
 
+# Sinatra error handling code randomly fails with Runtime Error. Monkey patch
+# the bad method.
```

Comment:
This is a temporary band-aid. We need to file an issue or something to make sure we drill down to what's actually happen and how to really fix it.